scripts/local-deploy.ts

```diff
@@ -212,12 +214,54 @@ async function main() {
       DefaultPartnerConfiguration.address
     );
 
+    const { contract: PartnerOneConfiguration } =
+      await deployContract<PartnerConfiguration>('PartnerConfiguration', {
+        accessControl: RegistrarAccessControlContract.address,
+        minLength: BigNumber.from(4),
+        maxLength: BigNumber.from(25),
+        minDuration: BigNumber.from(3),
+        maxDuration: BigNumber.from(8),
+        feePercentage: ZERO_FEE_PERCENTAGE,
+        discount: BigNumber.from(0),
+        minCommitmentAge: 0,
+      });
+
+    console.log('PartnerOneConfiguration:', PartnerOneConfiguration.address);
+
+    const { contract: PartnerTwoConfiguration } =
+      await deployContract<PartnerConfiguration>('PartnerConfiguration', {
+        accessControl: RegistrarAccessControlContract.address,
+        minLength: BigNumber.from(3),
+        maxLength: BigNumber.from(10),
+        minDuration: BigNumber.from(2),
+        maxDuration: BigNumber.from(7),
+        feePercentage: HALF_FEE_PERCENTAGE,
+        discount: HALF_DISCOUNT_PERCENTAGE,
+        minCommitmentAge: 1,
```

Comment:
1 second?

Reply:
we can change it later tho

Reply:
The main aim is just to trigger the commitment required behaviour, and since it's for local testing, wouldn't want to waste too much time waiting...